meta-xt-control-domain/recipes-guest/doma/files/doma-create.service

```diff
@@ -0,0 +1,39 @@
+# NOTE 1
+# Pay attention that corresponding part for
+# Dom0's backend-ready@block.service is
+# DomD's block-up-notification.service.
+# NOTE 2
+# If your product needs additional dependency, consider to use
+# systemd drop-in units instead of modifications in this file.
+# NOTE 3
+# Dependencies (After=, Requires=) cannot be reset to an empty list,
+# so dependencies can only be added in drop-ins. If you want to
+# remove dependencies, you have to override the entire unit.
+# NOTE 4
+# Pay attention, that this file starts and pauses the DomA right away.
+# This is done by intention, as we need to start QEMU in DomD before
+# unpausing the guest domain.
+# NOTE 5
+# This service depends on 'doma-restart-monitor.service', as when that
+# service fails, it is intended to restart DomA.
+
+[Unit]
+Description=Android VM creator service
+
+Requires=backend-ready@block.service doma-restart-monitor.service qemu-restart-monitor.service
+After=backend-ready@block.service doma-restart-monitor.service qemu-restart-monitor.service
+
+[Service]
+Type=simple
+
+ExecStart=/usr/lib/xen/bin/doma-create-ExecStart.sh
+
+ExecStopPost=/usr/lib/xen/bin/doma-create-ExecStopPost.sh
+
+Restart=always
+RemainAfterExit=true
```

Comment:
> RemainAfterExit=
>Takes a boolean value that specifies whether the service shall be considered active even when all its processes exited.

Is this a correct option for this service? I have no idea how it should work with "Restart=always"

Reply:
Hi @lorc, Here it is totally valid. It will remain valid even after the squash of doma-create and doma-unpause services.

The script of this service does something, and then, at some point, the script finishes and exits. Still, the service should remain active until DomA is running, as we are interested in the execution of the ExecStopPost in case if user calls "systemctl stop doma-create".

Reply:
Hi @lorc, regarding this point I find nothing to do, and I'm pretty convinced in a made conclusion. Thus, I'll close this conversation.

Please, reopen it if there are still open questions.